src/components/container/RestingHeartRateCalendar/RestingHeartRateCalendar.tsx

```diff
@@ -10,12 +10,14 @@ import { queryDailyData, DailyDataType } from '../../../helpers/query-daily-data
 
 type HeartRateMap = { [key: string]: number };
 
-export type RestingHeartRateCalendarPreviewState = "WithData" | "NoData" | "Loading";
+export type RestingHeartRateCalendarPreviewState = "WithData" | "NoData" | "Loading" | "Live";
```

Comment:
The absence of a preview state would signal Live. The value can be removed from RestingHeartRateCalendarPreviewState